.gitignore

```diff
@@ -8,3 +8,6 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+
+
+.idea/
```

Comment:
i tend to not check-in files that are specific to just my personal set-up... what do you think about dropping this to merge into `main`?

Reply:
agree